Update pgjwt documentation with link to Supabase JWT Handling Documen… (#40272)
supabase/supabase: apps/docs/content/guides/database/extensions/pgjwt.mdx

description: 'Encode and decode JWTs in PostgreSQL'
---

<Admonition type="note">

Supabase creates and handles JWT for you. It is built into the platform. **If you use Postgres version 15 or earlier**, you don't need the pgjwt extension, and it is safe to disable. For more information on how Supabase handles JWTs, read the [Supabase and JWTs documentation](/docs/guides/auth/jwts#supabase-and-jwts)

</Admonition>

<Admonition type="deprecation">

The `pgjwt` extension is deprecated in projects using Postgres 17. It continues to be supported in projects using Postgres 15, but will need to dropped before those projects are upgraded to Postgres 17. See the [Upgrading to Postgres 17 notes](/docs/guides/platform/upgrading#upgrading-to-postgres-17) for more information.

> 

Reviewer: ```suggestion
```

{/* supa-mdx-lint-disable-next-line Rule004ExcludeWords */}

> Note, Supabase creates and handles JWT for you. It is built into the platform so the pgjwt extension is not needed and is safe to disable. For more information on how Supabase handles JWT see our documentation here: [Supabase and JWTs](https://supabase.com/docs/guides/auth/jwts#supabase-and-jwts)

Reviewer: ```suggestion
<Admonition type="note">

Supabase creates and handles JWT for you. It is built into the platform. You don't need the pgjwt extension, and it is safe to disable. For more information on how Supabase handles JWT, read the [Supabase and JWTs documentation](/docs/guides/auth/jwts#supabase-and-jwts)

</Admonition>
```
